lms/djangoapps/commerce/migrations/0001_data__add_ecommerce_service_user.py

```diff
@@ -21,6 +23,11 @@ def backwards(apps, schema_editor):
 
 class Migration(migrations.Migration):
 
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('user_api', '0002_retirementstate_userretirementstatus'),
```

Comment:
**Note to Reviewer:** This must be a new dependency required for creating a user.